docs/guides/operator-private-gcp-confidential-space.md

```diff
@@ -0,0 +1,405 @@
+
+- Integration environment: see [Integration Deployment](#integration-deployment)
+- Production environment: see [Production Deployment](#production-deployment)
+
+>NOTE: As a best practice, we recommend that you test and verify your implementation in the integration environment before deploying in the production environment.
+
+### Integration Deployment
+
+To deploy a new UID2 Operator in the GCP Confidential Space Enclave into the integration environment, you'll need to create a VM instance.
+
+Follow these steps:
+
+1. [Update the Script with Valid Values](#update-the-script-with-valid-values)
+1. [Run the Script](#run-the-script)
+1. [Test Using the Health Check Endpoint](#test-using-the-health-check-endpoint)
+
+#### Update the Script with Valid Values
+
+Update the example script, using your own valid values instead of the placeholder values shown in the table.
+
+The following example uses placeholder values.
+
+```
+$ gcloud compute instances create {INSTANCE_NAME} \
+  --confidential-compute \
+  --shielded-secure-boot \
+  --maintenance-policy Terminate \
+  --scopes cloud-platform \
+  --image-project confidential-space-images \
+  --image-family confidential-space \
+  --service-account {SERVICE_ACCOUNT} \
+  --metadata ^~^tee-image-reference=ghcr.io/iabtechlab/uid2-operator@sha256:{IMAGE_SHA}~tee-restart-policy=Never~tee-env-DEPLOYMENT_ENVIRONMENT=integ~tee-env-API_TOKEN={API_TOKEN}
+```
+
+(**GWH_YS10 URL update please for the above URL**)
+
+(**GWH_YS11 Do we need the \ at the end of each line in the above?**)
+
+Placeholder values are defined in the following table.
+
+| Placeholder | Actual Value |
+| :--- | :--- |
+| `{INSTANCE_NAME}` | Your own valid VM name. |
+| `{SERVICE_ACCOUNT}` | The value you provided during account creation, in this format: `{SERVICE_ACCOUNT_NAME}@{PROJECT_ID}.iam.gserviceaccount.com`.<br/>For details, see [Confidential Space Account Setup](#confidential-space-account-setup) (Step 4). |
+| `{IMAGE_SHA}` | The SHA for your UID2 operator image digest, received during your UID2 account setup.<br/>For details, see [UID2 Operator Account Setup](#uid2-operator-account-setup). |
+| `{API_TOKEN}` | Your UID2 `api_token` value, received during your UID2 account setup, which identifies the private operator service.<br/>For details, see [UID2 Operator Account Setup](#uid2-operator-account-setup). |
+
+#### Run the Script
+
+When you've replaced the placeholder values with valid values, run the script.
+
+#### Test Using the Health Check Endpoint
+
+Call the health check endpoint to test the health of your implementation. The expected result is an `OK` response.
+
+```
+Get the public Ip of the deployed instance
+gcloud compute instances describe {INSTANCE_NAME} \
+    --format='get(networkInterfaces[0].accessConfigs[0].natIP)'
+health check
+  http://{IP}:8080/ops/healthcheck
+```
+
+(**GWH_YS12 Is the health check valid for both integration and production environments?**)
```

Comment:
Yes. Integ and Prod are basically the same except the endpoints and credentials.